Ginger/GingerCore/Drivers/JavaDriverLib/JavaDriver.cs

```diff
@@ -1907,8 +1938,69 @@ List<AppWindow> IWindowExplorer.GetAppWindows()
             return list;
         }
 
-        List<ElementInfo> IWindowExplorer.GetVisibleControls(List<eElementType> filteredElementType, ObservableList<ElementInfo> foundElementsList = null,bool isPOMLearn = false)
+
+        private void GetWidgetsElementList(List<eElementType> selectedElementTypesList, ObservableList<ElementInfo> elementInfoList, string currentFramePath)
         {
+            var javaElementInfo = new JavaElementInfo()
+            {
+                XPath = currentFramePath
+            };
+
+            if (!isBrowserElementLearned)
```

Comment:
What is the purpose of this flag ?  If there are more than 1 browsers ? But what if there 2 widgets screen open and we found wrong one 1st. I think instead of flag, we should check if browser is in focus or some other attribute